src/gsi/read_radar.f90

```diff
@@ -3774,16 +3878,17 @@ subroutine read_radar_l2rw(ndata,nodata,lunout,obstype,sis,nobs,hgtl_full)
   nsuper2_in=0
   nsuper2_kept=0
   ntdrvr_thin2=0
-  maxout=0
-  maxdata=0
-  isort=0
   icntpnt=0
   nout=0
   if(loop==0) outmessage='level 2 superobs:'
   rmesh=radar_rmesh
   zmesh=radar_zmesh
   nlevz=nint(16000._r_kind/zmesh)
   xmesh=rmesh
+  pmot=radar_pmot
```

Comment:
As the code was written, the rmesh, zmesh,.. values were taken from radar_rmesh, radar_... values.  So I did the same thing with the pmot values.  It was not clear to me that the pmot_conv, ..._conv values would be defined for these particular observations.  It would be better to use the *_conv values in the convinfo file since it would not require a change to the code to change the values.  I do not completely understand the read_radar.f90 code.  :)